kubernetes/apps/authentication/zitadel/app/helmrelease.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: zitadel
+  namespace: authentication
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: zitadel
+      version: 8.13.1
+      sourceRef:
+        kind: HelmRepository
+        name: zitadel
+        namespace: flux-system
+      interval: 5m
+  values:
+    image:
+      tag: v3.0.3
+    configmapConfig:
+      FirstInstance:
+        Org:
+          Machine:
+            Machine:
+              Username: zitadel-iac-admin-sa
+              Name: Admin
```

Comment:
Pushing the generated secret is tbd in a follow-up PR (because idk what it will generate to)